src/libsyntax/json.rs

```diff
@@ -90,6 +90,14 @@ impl Emitter for JsonEmitter {
             panic!("failed to print diagnostics: {:?}", e);
         }
     }
+
+    fn maybe_emit_json_directive(&mut self, directive: String) {
+        let data = Directive { directive };
```

Comment:
```suggestion
        let data = Directive { unstable_directive };
```

To reduce likelyhood of people to start depending on them. It may be nice to keep directives behind a feature gate in the near future too.